.mailmap

```diff
@@ -397,3 +397,7 @@ Gagandeep Singh <singh.23@iitj.ac.in> MathLover <36567889+czgdp1807@users.norepl
 S.Y. Lee <sylee957@gmail.com> sylee957 <sylee957@gmail.com>
 Yatna Verma <yatnavermaa@gmail.com> yatna <yatnavermaa@gmail.com>
 Jared Lumpe <mjlumpe@gmail.com> Michael Jared Lumpe <mjlumpe@gmail.com>
+Austin Palmer <ap4000@nyu.edu> austin <ap4000@nyu.edu>
+Marduk Bolaños <mardukbp@mac.com> mardukbp <mardukbp@mac.com>
+amsuhane <ayushsuhane99@iitkgp.ac.in> amsuhane <“ayushsuhane99@iitkgp.ac.in”>
+RituRajSingh878 <RituRajSingh878@gmail.com> Ritu Raj Singh <37741324+RituRajSingh878@users.noreply.github.com>
```

Comment:
Is `RituRajSingh878` how you want it to appear in the AUTHORS? I would suggest

```
Ritu Raj Singh <RituRajSingh878@gmail.com> RituRajSingh878 <RituRajSingh878@gmail.com>
Ritu Raj Singh <RituRajSingh878@gmail.com> Ritu Raj Singh <37741324+RituRajSingh878@users.noreply.github.com>
``` 

But it's up to you which name/email combo you want to appear.

Reply:
thanks for the suggestion.